mkdocs.yml

```diff
@@ -1,6 +1,7 @@
 # yaml-language-server: $schema=https://squidfunk.github.io/mkdocs-material/schema.json
 
 site_name: Automated Research Assistant
+site_url: 'https://autoresearch.github.io/autora/'
```

Comment:
site_url: 'https://autoresearch.github.io/'

Reply:
I'm not sure that that's the right thing to do – it conflicts with the documentation on https://www.mkdocs.org/user-guide/configuration/#site_url, so I think if we do it, we risk breaking something else later. 
I've added a suggestion to fix the links using the other parameters available on the `page` object